CONTRIBUTING.md

```diff
@@ -5,3 +5,69 @@ We love improvements to our tools! EDGI has general [guidelines for contributing
 ## Submitting Web Monitoring Issues
 
 Issues that are project-wide, or relate heavily to the interaction between different components, should be added to our [Web Monitoring issue queue](https://github.com/edgi-govdata-archiving/web-monitoring/issues). Component-specific issues should be added to their respective repository.
+
+The following are recommended code styling and best practices for the web-monitoring-ui repository. We also have best practices for related to all the [web-monitoring] project (https://github.com/edgi-govdata-archiving/web-monitoring/blob/master/CONTRIBUTING.md) repo.
+
+## UI - Support
+### Browser Support:
+Last two of every major browser (Chrome, Safari, Firefox, Edge) except IE
```

Comment:
I’ve been trying to stick to a rule of two blank lines after text/before a heading and one blank line after headings, which is by no means critical, but I think it aids reading. I guess we didn’t talk about Markdown style at all :P

(Maybe we should add markdown linting?)

Reply:
Happy to continue with that convention. No, we didn't talk about Markdown style, better late than never. :)
Markdown linting is a good idea, I think ESLint may do that.